src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,9 @@
+import PermanentDrawerLeft from '../../components';
+
+const HomePage = () => (
+  <div>
+    <PermanentDrawerLeft />
+  </div>

```

Comment:
why u need `div` here?

```suggestion
    <PermanentDrawerLeft />
```